beesdoo_product/models/beesdoo_product.py

```diff
@@ -348,6 +358,13 @@ def _inverse_purchase_price(self):
                     _("No Vendor defined for product '%s'") % product.name
                 )
 
+    @api.multi
+    @api.depends("purchase_price", "seller_ids")
+    def _compute_purchase_price_write_date(self):
+        for product in self:
+            supplierinfo = product._get_main_supplier_info()
```

Comment:
`supplier_info` looks better maybe ? (really really nitpicking here :smile: )

Reply:
The model is called `product.supplierinfo` :)

Reply:
ah ok, perfect then !